lib/bindings/AMQPBinding.js

```diff
@@ -190,8 +190,6 @@ function start(callback) {
                                 numRetried++;
                                 return setTimeout(createConnection, retryTime * 1000);
                             }
-                        } else {
```

Comment:
CC: @fiqare-secmotic

Curiously this is contrary to one of the rules from fiQare project, see https://github.com/telefonicaid/iotagent-ul/pull/411#discussion_r358409771. We enforce one thing (to remove empty else brarnches) and the opposite (to enforce else branch even when empty) at the same time :)

@jason-fox @fiqare-secmotic what do you think?

Reply:
This is an `if...else...` construct not an `if...else if...` construct.  

The `return` within the `else` clause is totally unnecessary.

Reply:
Hi! yes, as @jason-fox says, it is not an  `if...else if` construct

Reply:
Just to check if I'm understanding it correctly...

The MISRA and CERT rules cited at https://github.com/telefonicaid/iotagent-ul/pull/411#discussion_r364121752, prescribe the inclusion of final "else" in "if... else if..." structures *but not* prescribe the use of "else" in simple "if" structured.

@fiqare-secmotic, could you confirm my understanding is correct?

Reply:
> @fiqare-secmotic, could you confirm my understanding is correct?

Any news?

Reply:
@fgalan yes, your understanding is correct.